uber/tests/conftest.py

```diff
@@ -1,24 +1,49 @@
 from uber.common import *
```

Comment:
There's not that much to say about the unit test fixes, but most of the changes fall into one of three categories:
- accounting for the ``c`` refactor; since we now have a global object that holds all of our config, we no longer need to mess around with module globals
- modifying or removing existing test cases because the behavior they're asserting is no longer part of Uber/RAMS (side note: is it "RAMS" or "Rams"?)
- added some default config to override whatever is in the config files so that we can write unit tests without worrying about whether ``c.ARCADE`` is actually defined, etc

I'll be happy to answer any specific questions, and we can go over unit tests in general whenever.